dask_kubernetes/core.py

```diff
@@ -20,6 +20,8 @@
 
 logger = logging.getLogger(__name__)
 
+DEFAULT_WORKER_TEMPLATE_PATH = "/etc/dask/kubernetes/worker-template.yaml"
```

Comment:
We could also use `~/.dask/kubernetes/...` which is where we currently place our config files.  I think that this directory is configurable by setting the `DASK_CONFIG` environment variable (we should probably document this better).

Reply:
The common way would be to lookup in that order:

- `/etc/dask/kubernetes/worker-template.yaml`
- `~/.dask/kubernetes/worker-template.yaml`
- `DASK_KUBERNETES_WORKER_TEMPLATE_PATH`

and take the last file that exists.

Reply:
I'm fine with adopting that convention.  Should we eventually adopt this more broadly throughout the project?

Reply:
Which other configuration files do you have in mind? You mean dask in general or just dask-kubernetes?

Reply:
BTW, nowadays I think that using `~/.config/dask/` would be better than `~/.dask/` but if we switch we should be careful about backward dealing with compat.

Reply:
I was referring to dask in general.

Hrm, looking at my own `~/.config` directory I now agree.  I'll raise an issue upstream.